server/src/immich/api-v1/asset/asset.service.ts

```diff
@@ -80,10 +81,15 @@ export class AssetService {
       const libraryId = await this.getLibraryId(auth, dto.libraryId);
       await this.access.requirePermission(auth, Permission.ASSET_UPLOAD, libraryId);
       this.requireQuota(auth, file.size);
+      await this.fixModifiedAtTimestamp(file, dto);
       if (livePhotoFile) {
+        await this.fixModifiedAtTimestamp(livePhotoFile, dto);
```

Comment:
These two can just go inside of the create method instead. Probably can move the sidecar one in there as well.

Reply:
Sounds good, I've moved all timestamp updating into the `create` method